### ISSUE TITLE
Create an index for `lib_siglum` on holdings and sources

### ISSUE THREAD

**Issue by ahankinson** [rism-digital/muscat#1267]
To help look up the correct number of sources for institutions, we can use the `lib_siglum` field on holdings and sources. However, without an index on these fields the query to do this takes a very long time. With an index, the query takes a matter of seconds.

So could we please have an index added to the `lib_siglum` fields for these two tables? (`holdings` and `sources`)? Preferably, this would be added to the production server as soon as possible.

**Comment by HirschSt** [comment]
Probably you mean a database-index (not solr) with a new migration, yes?

**Comment by xhero** [comment]
Yes! Since it is relatively small and does not touch the schema, I was thinking to do it in a minor release like 8.1.3.
